docs/refarch/boundedContexts/scheduling/index.md

```diff
@@ -1,23 +1,133 @@
-# {name} BC
+# Scheduling BC
 
-{overview}
+Multiple processes and events in BCs across the Mojaloop Switch platform require functionality to trigger at specific times, or in accordance with a schedule.  In order to support this requirement across the Switch, and avoid building scheduling functionality into multiple BCs, a single Scheduling BC will be introduced and implemented for the Switch platform.
+
+In order to schedule a process or event, a Client BC submits a request to the Scheduling BC to create a reminder for a specific time or time schedule.  The Scheduling BC maintains a schedule of all of the reminders that it receives, and, as the time for reminders is reached, it sends notification of the reminder to the appropriate Client BC.
+
+In addition, the Scheduling BC will also provide services to the Switch to enable Client BCs and Switch Admins to manage reminders that are set within the Scheduling BC.
 
 ## Terms
 
-Terms with specific and commonly accepted meaning within the Bounded Context in which they are used.
+The following common term/s are in use in this BC:
 
 | Term | Description |
 |---|---|
-| Term1 | Description1 |
+| Client BC | Any other BC making use of the Scheduling BC Services |
 
 ## Use Cases
 
-### Perform Transfer (universal mode)
+***Note:*** *An "Update Reminder" UC has not been provided. The recommended course of action to follow is to Delete and Create a new Reminder.*
```

Comment:
Would we lose any auditing/associated data when deleting a reminder vs doing an update _(at least that we care about)_?